### DISCUSSION TITLE
Is there a plan to divide intersection-observer core and frameworks workaround into packages by using workspaces, like monorepo technology?

### DISCUSSION THREAD

**Discussion by Albert-cord** [thebuilder/react-intersection-observer#624] [General]
So we can write other framework workarounds like vue, solid, etc, with the same core.
If the author approves it, I will contribute some pr when I'm free.

**Comment by thebuilder** [comment]
I have been thinking about it. Could make sense to expand it to other frameworks, using the same core. 

The repo a recently converted to pnpm workspaces, but there's some work in actually publishing different packages.
